src/Microsoft.VisualStudio.Services.Agent/Blob/BlobStoreUtils.cs

```diff
@@ -125,7 +125,7 @@ private static async Task<List<BlobFileInfo>> GenerateHashes(IReadOnlyList<strin
                         {
                             Path = itemPath,
                             Node = dedupNode,
-                            Success = dedupNode != null
+                            Success = true
```

Comment:
This seems like an unrealated functional change

Reply:
dedupNode is a struct, so comparing against null gives a warning. double checked the implementation, the semantics of this api is that if it fails, it throws, which is already handled.